cdk-lib/viewer-stacks/viewer-vpc-stack.ts

```diff
@@ -13,72 +11,72 @@ export interface ViewerVpcStackProps extends StackProps {
+            subnetIds: this.vpc.privateSubnets.map(obj => obj.subnetId),
+            transitGatewayId: props.captureTgw.attrId,
+            vpcId: this.vpc.vpcId,
 
-        // the properties below are optional
-        options: {
-            "DnsSupport": "enable"
-        },
-    });
+            // the properties below are optional
+            options: {
+                'DnsSupport': 'enable'
+            },
+        });
 
-    // Make sure that hosts inside the Viewer VPC's private subnets know how to talk to the Capture VPC
-    this.vpc.privateSubnets.forEach((value, index) => {
+        // Make sure that hosts inside the Viewer VPC's private subnets know how to talk to the Capture VPC
+        this.vpc.privateSubnets.forEach((value, index) => {
 
-        const subnet = value as ec2.Subnet;
-        // subnet.addRoute(`ToCaptureVpc${index + 1}`, {
-        //     routerId: props.captureTgw.attrId,
-        //     routerType: ec2.RouterType.TRANSIT_GATEWAY,
-        //     destinationCidrBlock: props.captureVpc.vpcCidrBlock,
-        // })
-        const route = new ec2.CfnRoute(this, `ToCaptureVpc${index + 1}`, {
-            routeTableId: subnet.routeTable.routeTableId,
-            transitGatewayId: props.captureTgw.attrId,
-            destinationCidrBlock: props.captureVpc.vpcCidrBlock
+            const subnet = value as ec2.Subnet;
+            // subnet.addRoute(`ToCaptureVpc${index + 1}`, {
+            //     routerId: props.captureTgw.attrId,
+            //     routerType: ec2.RouterType.TRANSIT_GATEWAY,
+            //     destinationCidrBlock: props.captureVpc.vpcCidrBlock,
+            // })
+            const route = new ec2.CfnRoute(this, `ToCaptureVpc${index + 1}`, {
+                routeTableId: subnet.routeTable.routeTableId,
+                transitGatewayId: props.captureTgw.attrId,
+                destinationCidrBlock: props.captureVpc.vpcCidrBlock
+            });
+            route.addDependency(tgwAttachment);
         });
-        route.addDependency(tgwAttachment);
-    });
 
-    // Make sure that return traffic back to the Viewer VPC has a way to get there
-    props.captureVpc.privateSubnets.forEach((value, index) => {
-        const subnet = value as ec2.Subnet;
-        // subnet.addRoute(`ToViewerVpc${index + 1}`, {
-        //     routerId: props.captureTgw.attrId,
-        //     routerType: ec2.RouterType.TRANSIT_GATEWAY,
-        //     destinationCidrBlock: this.vpc.vpcCidrBlock,
-        // })
+        // Make sure that return traffic back to the Viewer VPC has a way to get there
+        props.captureVpc.privateSubnets.forEach((value, index) => {
+            const subnet = value as ec2.Subnet;
+            // subnet.addRoute(`ToViewerVpc${index + 1}`, {
+            //     routerId: props.captureTgw.attrId,
+            //     routerType: ec2.RouterType.TRANSIT_GATEWAY,
+            //     destinationCidrBlock: this.vpc.vpcCidrBlock,
+            // })
```

Comment:
Looks like I forgot to clean up some test code in a previous PR.  Will remove in my next one.